### PR TITLE
Prepare the release 4.0

### PR DESCRIPTION
**Pull request by LaurentPetard**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
+# 4.0.0
```
Comment (LaurentPetard) [inline CHANGELOG.md]: Set the release date before merging

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
+# 4.0.0 (2019-02-08)
```
Comment (LaurentPetard) [inline CHANGELOG.md:1]: Change the date at the release

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+- Get a list of reference entity attributes
+- Upsert a single reference entity attribute
+- Get a single reference entity attribute option
+- Get a list of reference entity attribute options
+- Upsert a single reference entity attribute option
+- Download a reference entity media file
+- Create a reference entity media file
+
+## BC breaks
+
```
Comment (NolwennP) [inline CHANGELOG.md:25]: Can we say that these BC will only occur if the user extended those interfaces? Or something like this?

**Review by ahocquard — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (ahocquard) [reply CHANGELOG.md:25]: I think it's useless, it's pretty obvious for developers. Also, we did a BC when getting an asset (for DX, issue raised by someone).

**Pull request merged**